packages/grid/_modules_/grid/models/api/gridCsvExportApi.ts

```diff
@@ -1,14 +1,20 @@
+import { GridExportCsvOptions } from '../gridExport';
+
 /**
- * The csv export API interface that is available in the grid [[apiRef]].
+ * The CSV export API interface that is available in the grid [[apiRef]].
  */
 export interface GridCsvExportApi {
   /**
-   * Returns the grid data formatted as CSV.
-   * @returns {string} The data as CSV.
+   * Returns the grid data as a CSV string.
+   * This method is used internally by `exportDataAsCsv`.
+   * @params GridExportCsvOptions The options to apply on the export.
```

Comment:
https://jsdoc.app/tags-param.html#names-types-and-descriptions

```suggestion
   * @param {GridExportCsvOptions} options The options to apply on the export.
```